manifests/charts/istio-cni/values.yaml

```diff
@@ -28,6 +28,9 @@ cni:
   # Custom annotations on pod level, if you need them
   podAnnotations: {}
 
+  # Custom Labels on pod level, if you need them
+  podLabels: {}
```

Comment:
should we define `k8s-app: istio-cni-node` here?

Reply:
Would be a solution, if you want to have this configurable, but for annotations you have not done it in the chart.

If you want to have it, I can add that. Otherwise I would also be happy to merge this as is.